staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -300,34 +300,39 @@ func TestUnconditionalDelete(t *testing.T) {
 	key, storedObj := testPropogateStore(ctx, t, store, &example.Pod{ObjectMeta: metav1.ObjectMeta{Name: "foo"}})
 
 	tests := []struct {
+		name string
 		key               string
 		expectedObj       *example.Pod
 		expectNotFoundErr bool
-	}{{ // test unconditional delete on existing key
+	}{{
+		name: "test unconditional delete on existing key",
 		key:               key,
 		expectedObj:       storedObj,
 		expectNotFoundErr: false,
-	}, { // test unconditional delete on non-existing key
+	}, {
+		name: "test unconditional delete on non-existing key",
```

Comment:
names passed to `t.Run` show up in test grid and explode the left-hand side width, right? might need to trim verbosity a little/a lot

Reply:
@liggitt I'm not extremely familiar with TestGrid rendering, sorry - is there a guideline or length limit I should target? I can take a stab at shortening the names.

Reply:
https://testgrid.k8s.io/presubmits-kubernetes-blocking#pull-kubernetes-unit&include-filter-by-regex=k8s.io%2Fapiserver%2Fpkg%2Fstorage%2Fetcd3 is the ones we currently have

looks like we already have some ~verbose ones, so maybe this one is fine, but some of the ones below that tell little stories should be trimmed ("test GetToList on existing key with minimum resource version set to current resource version, match=minimum")

Reply:
Yeah, looks like the same comment -> name approach was taken in e.g. `TestList` conversion. I'll add a commit to shorten some of these where I can.